sdks/python/apache_beam/examples/cookbook/coders_test.py

```diff
@@ -78,24 +94,23 @@ def test_compute_points(self):
   @pytest.mark.examples_postcommit
   def test_coders_output_files_on_small_input(self):
     test_pipeline = TestPipeline(is_integration_test=True)
-
     # Setup the files with expected content.
-    temp_folder = tempfile.mkdtemp()
-    self.create_content_input_file(
-        os.path.join(temp_folder, 'input.txt'),
-        '\n'.join(map(json.dumps, self.SAMPLE_RECORDS)))
-    extra_opts = {
-        'input': '%s/input.txt' % temp_folder,
-        'output': os.path.join(temp_folder, 'result')
-    }
+    OUTPUT_FILE_DIR = \
```

Comment:
With this change, are we going to access GCS in unit test (since this is a unit test file) ?

Reply:
Yes, these tests were failing in Dataflow because of the use of local temp directories, do you think is a better option to move them to a different file?

Reply:
I think unit tests should be light-weight and not require credentials to apache-beam-testing. integration tests may need that access. 

We can separate integration tests into  _it_test files, or mark them a postcommits within the same test file. My preference is the former but I think we have instances of both, so no strong opinion on that.

Reply:
I will move them to it_test files

Reply:
Done!